content/pages/disability-benefits/claims-appeal.md

```diff
@@ -23,66 +23,71 @@ When the VBA Regional Office (RO) receives your Notice of Disagreement (NOD), yo
+      When you receive a copy of the Statement of the Case in the mail, you’ll also get a VA Form 9. If you disagree with the Statement of the Case and want to continue your appeal to the Board of Veterans’ Appeals, you must return VA Form 9 within 60 days.
     </p>
   </li>
   <li class="list-group-item">
-    <h4>(Optional) RO prepares the Supplemental Statement of the Case (SSOC)</h4>
+    <h4>The Veterans Benefits Administration prepare a new Statement of the Case (optional visual treatment)</h4>
     <p>
-      You can submit new evidence for your appeal at any time in the process. Each time you do, the RO will need to make a decision or prepare an SSOC if the RO can’t grant all or part of your appeal. This may add time to your appeal, but may make your appeal more successful if the evidence supports your argument. This step can happen at any time after the RO prepares the SOC.
+      You can submit new evidence for your appeal at any time in the process. If you submit new evidence after you receive a Statement of the Case, the Decision Review Officer will need to write a new Statement of the Case before transferring your case to the Board of Veterans’ Appeals. Once your appeal is transferred, new evidence can be sent directly to the Board and will not be reviewed by the Veterans Benefits Administration.
     </p>
   </li>
   <li class="list-group-item">
-    <h4>RO certifies your appeal to the Board</h4>
+    <h4>Your appeal is sent to the Board</h4>
     <p>
-      The RO will prepare your appeal and send it to the Board for review. Your appeal will wait in line until the Board is able to review it.
+    The Decision Review Officer will finish their review and transfer your case to the Board of Veterans’ Appeals. The Board reviews cases in the order they are received as determined by the date of the VA Form 9. A judge will begin work on your appeal when it is among the oldest appeals that are ready for their review.
     </p>
-  </li>
-  <li class="list-group-item">
-    <h4>The Board activates your appeal</h4>
     <p>
-      The Board now has control of your appeal. Lawyers who are experts in Veterans law and reviewing benefit claims will review it. The review prepares your appeal for a Veterans Law Judge who will make a decision. In most cases, your appeal is reviewed in the order it was received. In some cases, an appeal will be moved to the front of the line—this happens if you’re facing severe financial hardship, over age 75, or have a serious illness.
+    If you are suffering a serious illness or are in financial distress, or for other sufficient cause, you can apply to have your appeal Advanced on Docket. If you are older than 75, your appeal will receive this status automatically. Advanced on Docket appeals are prioritized so that they are always at the front of the line. Learn how to request Advanced on Docket status.
```

Comment:
This link needs to be updated, once the Advanced on Docket page is set up.

Reply:
Good call. If you write up a ticket for this, you can go ahead and merge this PR before that page is set up. :+1:

Reply:
The PR to set up that page may lap this PR, is: `/disability-benefits/claims-appeal/request-a-priority-review`, the appropriate link to include here @gnakm ?